Data/Aeson/Types/FromJSON.hs

```diff
@@ -343,18 +343,17 @@ genericLiftParseJSON opts pj pjl = fmap to1 . gParseJSON opts (From1Args pj pjl)
 --
 -- data Coord = Coord { x :: Double, y :: Double } deriving 'Generic'
 --
--- instance FromJSON Coord
+-- instance 'FromJSON' Coord
 -- @
 --
--- If @DefaultSignatures@ doesn't give exactly the results you want,
+-- If the default implementation doesn't give exactly the results you want,
 -- you can customize the generic decoding with only a tiny amount of
 -- effort, using 'genericParseJSON' with your preferred 'Options':
 --
 -- @
--- instance FromJSON Coord where
---     parseJSON = 'genericParseJSON' 'defaultOptions'
+-- instance 'FromJSON' Coord where
+--     'parseJSON' = 'genericParseJSON' customOptions
```

Comment:
Maybe have an example like `defaultOptions { sumEncoding = ObjectWithSingleField }` instead? Otherwise we don't explain how to construct `Options`. It also discourages usage of the Options constructor.